src/plone/restapi/services/auth/login.py

```diff
@@ -73,6 +75,8 @@ def _find_userfolder(self, userid):
             uf = getToolByName(uf_parent, 'acl_users')
             if uf:
                 info = uf._verifyUser(uf.plugins, login=userid)
+            if IPloneSiteRoot.providedBy(uf_parent):
```

Comment:
@tisto it is a pointer to the current node in the path; I've renamed it to context.